services/src/main/java/org/keycloak/authentication/authenticators/x509/AbstractX509ClientCertificateAuthenticatorFactory.java

```diff
@@ -130,7 +136,7 @@
         userMapperList.setOptions(mapperTypes);
 
         ProviderConfigProperty attributeOrPropertyValue = new ProviderConfigProperty();
-        attributeOrPropertyValue.setType(STRING_TYPE);
+        attributeOrPropertyValue.setType(MULTIVALUED_STRING_TYPE);
```

Comment:
Please also update the tooltip and add the information that multiple values are relevant just in case that mapping name contains multiple values. Which is currently just for newly added "Certificate Serial Number and IssuerDN". Thanks

Reply:
Good catch.
Updated.